plugins/source/aws/resources/services/kms/keys.go

```diff
@@ -81,6 +81,15 @@ func getKey(ctx context.Context, meta schema.ClientMeta, resource *schema.Resour
 		options.Region = cl.Region
 	})
 	if err != nil {
+		if client.IsAWSError(err, "AccessDenied") || client.IsAWSError(err, "AccessDeniedException") {
+			resource.Item = &types.KeyMetadata{
+				Arn:   item.KeyArn,
+				KeyId: item.KeyId,
+			}
+			cl.Logger().Warn().Err(err).Msg("metadata retrieved from ListKeys will still be persisted")
```

Comment:
I wonder if we really need a warning for every instance; I think one warning, or even just a mention in the documentation of this table, should suffice

Reply:
Users could have access to some keys but not others... and this follows the pattern we use in Lambda

Reply:
I'm just worried it's going to be a bit noisy if users have many keys. But it's okay, we can start with this and see if it's an issue for anyone